src/components/IndividualArticle/IndividualArticle.jsx

```diff
@@ -4,19 +4,57 @@ import { useParams } from "react-router-dom";
 import { fetchSingleArticle } from "../../api";
 
 const IndividualArticle = () => {
-  const [article, setArticle] = useState([]);
+  const [article, setArticle] = useState("");
   const [error, setError] = useState("");
   const [isLoading, setIsLoading] = useState(false);
   const { article_id } = useParams();
 
   useEffect(() => {
-    fetchSingleArticle(article_id).then((data) => {
-      console.log(data);
-      return data;
-    });
-  }, [article_id]);
+    setIsLoading(true);
+    fetchSingleArticle(article_id)
+      .then(({ article }) => {
+        console.log(article);
 
-  return <div></div>;
+        setArticle(article);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        console.log(error);
+
+        setError(true);
+      });
+  }, []);
+
+  if (isLoading) return "Loading...";
+  if (error) {
+    return <p>Error</p>;
+  }
+
+  return (
+    <div>
+      <h3>{article.title}</h3>
+      <img
+        src={article.article_img_url}
+        alt={article.title}
+        style={{ width: "20em", height: "15em" }}
+      />
+      {/* <p>
+        <i>{article.created_at.split("T")[0]}</i>
+      </p> */}
+      <p>
+        Author:
+        <i>
+          <strong> {article.author}</strong>
+        </i>
+        <span>
+          , Topic:
+          <i>
+            <strong> {article.topic}</strong>
+          </i>
+        </span>
+      </p>
+    </div>
+  );
```

Comment:
Consider what else the object contains, what you'd like to show on the page and how you might show it.